src/redux/Provider.tsx

```diff
@@ -49,7 +49,8 @@ const ReduxProvider = ({ children, locale }) => {
         const reciterId =
           userPreferences[PreferenceGroup.AUDIO]?.reciter ||
           audioService.getSnapshot().context.reciterId;
-        audioService.send({ type: 'SET_INITIAL_CONTEXT', playbackRate, reciterId });
+        const { volume } = audioService.getSnapshot().context;
```

Comment:
I see `audioService.getSnapshot().context` is repeated multiple times for `volume`, `playbackRate` and `reciterId` so perhaps we should put it in a `const` and reuse it for all.